packages/open-next/src/core/routing/matcher.ts

```diff
@@ -149,7 +143,7 @@ export function addNextConfigHeaders(
     source,
     locale,
   } of configHeaders) {
-    const path = locale === false ? rawPath : localizedRawPath;
+    const path = locale === false ? event.rawPath : localizePath(event);
```

Comment:
The `localizePath` was done outside in order to avoid recomputing the same value for every value. 
Not that big of a deal but worth mentioning

Reply:
OOps, I missed that, I'll revert